pkg/reconciler/service/install.go

```diff
@@ -59,6 +59,13 @@ func (r *Install) Invoke(ctx context.Context, chartProvider chart.Provider, task
 			},
 			newClusterWideResourceInterceptor(),
 			&NamespaceInterceptor{},
+			&FinalizerInterceptor{
```

Comment:
Removing finalizers by Reconciler can cause issues for components other than  `logging`.  It would make sense to apply `FinalizerInterceptor`for all resource types.